cli/src/merge_tools/mod.rs

```diff
@@ -166,7 +168,7 @@ fn editor_args_from_settings(
     if let Some(args) = settings.config().get(key).optional()? {
         Ok(args)
     } else {
-        let default_editor = "meld";
+        let default_editor = BUILTIN_EDITOR_NAME;
```

Comment:
If I understand the change in this commit correctly, shouldn't we mention it in the changelog? And I think there's documentation to update as well.